user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md

```diff
@@ -0,0 +1,239 @@
+
+   ![Add Row Numeric Table](~/user-guide/images/VisioURLFeed_AddRowNumericTable.png)
+
+1. Enter the row's primary key [IDX], enter random numbers into the first three columns, and click *OK*.
+
+   > [!IMPORTANT]
+   > Do not use any special characters for the primary key, as they will interfere with the URL linking later in this tutorial.
+
+   ![Table input](~/user-guide/images/VisioURLFeed_AddRowNumericTableInput.png)
+
+1. Add at least one more row to the table.
+
+   ![Table rows](~/user-guide/images/VisioURLFeed_NumericTableRows.png)
+
+## Step 2: Create a new dashboard and add a 'Parameter feed' component and a 'State' component to display the numerical data from the Virtual Connector
+
+1. Open the *Dashboards* app.
+
+1. [Create a new dashboard](xref:Creating_a_completely_new_dashboard).
+
+1. If you are not yet in edit mode, then click *Start editing*.
+
+1. Add a [Parameter feed component](xref:DashboardParameterFeed).
+
+1. From the *Data* pane on the right, drag the *ELEMENTS* header onto the *Parameter feed* component to be used as data.
+
+1. From the *Data* pane on the right, add the necessary parameters:
+
+   1. Expand the *PARAMETERS* section.
+
+   1. Set *From* to "Protocol".
+
+   1. Set *Protocol* to "Skyline Generic Virtual Connector".
+
+   1. Use the filter to find the three numeric table parameters that you entered numbers for.
+
+   1. Drag each of these parameters onto the *Parameter feed* component to be used as data.
+
+   ![Parameters](~/user-guide/images/VisioURLFeed_ParameterDataForFeed.png)
+
+1. From the *Data* pane on the right, add a protocol filter:
+
+   1. Expand the *PROTOCOLS* section.
+
+   1. Search for the *Skyline Generic Virtual Connector*.
+
+   1. Drag the protocol onto the *Parameter feed* component to be used as a filter.
+
+   ![Protocols](~/user-guide/images/VisioURLFeed_ProtocolDataForFeed.png)
+
+1. Add a [State component](xref:DashboardState).
+
+1. Click the *State* component to select it.
+
+1. From the *Data* pane on the right, add the *Parameters* feed:
+
+   1. Expand the *FEEDS* section to find the items associated with the parameter feed.
+
+   1. Expand the *Parameter feed* component.
+
+   1. Drag the *Parameters* item onto the *State* component to be used as data.
+
+   ![Feeds](~/user-guide/images/VisioURLFeed_ParameterDataForState.png)
+
+1. In the *Layout* pane on the right, select the *LABELS* boxes to display the parameter name, the index, and the value.
+
+   ![Labels](~/user-guide/images/VisioURLFeed_StateLayoutOptions.png)
+
+1. Click *Stop editing* to exit edit mode.
+
+   At this point, you should be able to make manual parameter feed selections to control which parameter values are shown on the *State* component.
+
+   ![State component](~/user-guide/images/VisioURLFeed_DashboardManualResults.png)
+
+1. In the top-right corner of the screen, click the ellipsis button, and then click *Share*.
```

Comment:
Could you add a screenshot of this action as well?

Reply:
Done